GitUI/CommandsDialogs/FormMailMap.cs

```diff
@@ -138,7 +138,7 @@ private void FormMailMapLoad(object sender, EventArgs e)
                 return;
             }
 
-            MessageBox.Show(this, _mailmapOnlyInWorkingDirSupported.Text, _mailmapOnlyInWorkingDirSupportedCaption.Text);
+            MessageBox.Show(this, _mailmapOnlyInWorkingDirSupported.Text, _mailmapOnlyInWorkingDirSupportedCaption.Text, MessageBoxButtons.OK, MessageBoxIcon.Information);
```

Comment:
I think this should be consistent with GitUI/CommandsDialogs/FormGitIgnore.cs line:197 - which is an error.
Or at least a warning, not an info

Reply:
I think I will use error.